scripts/hdf5_dump.py

```diff
@@ -233,20 +235,20 @@ def print_header_dict(hdict, clock_speed_hz):
 
 
 def print_trigger_record_header(data_array, clock_speed_hz, k_list_components):
-    print_header_dict(unpack_header(data_array, "TriggerRecord Header"), clock_speed_hz)
+    print_header_dict(unpack_header(data_array, "TriggerRecord Header", 4), clock_speed_hz)
 
```

Comment:
I think it would be better to have something like `TRIGGER_RECORD_HEADER_VERSION = 4` at the top of this file and used here to improve clarity and any future updates.